scripts/prepdocs.sh

```diff
@@ -4,14 +4,6 @@
 
 echo 'Running "prepdocs.py"'
 
-AZURE_USE_AUTHENTICATION=$(azd env get-value AZURE_PUBLIC_NETWORK_ACCESS)
```

Comment:
prepdocs.ps1 was already missing this check, so we just remove it from sh and add it to prepdocs.py.